ui/src/main/java/edu/wpi/grip/ui/Main.java

```diff
@@ -20,6 +20,7 @@
 import com.google.inject.Guice;
 import com.google.inject.Injector;
 import com.google.inject.util.Modules;
+import com.sun.javafx.application.LauncherImpl;
```

Comment:
@JLLeitschuh Just for future reference, `com.sun.*` is fine, it's `sun.*` that should be avoided